src/RangeCalendar.js

```diff
@@ -322,6 +322,8 @@ const RangeCalendar = createReactClass({
       const minutes = startValue.minute();
       const second = startValue.second();
       let { disabledHours, disabledMinutes, disabledSeconds } = userSettingDisabledTime;
+      const oldDisabledMinutes = disabledMinutes ? disabledMinutes() : [];
```

Comment:
I don't think change the return of `disabledTime` is a good idea. We should call this `getEndDisableTime` when CalendarPart renders.

Reply:
It isn't change the return of disabledTime. This is for when the hour isn‘t equal，disabledMinutes in getEndDisableTime will return [].But disabledMinutes should return use's disabledMinutes。

Reply:
Hmm, I misunderstand the API.